src/components/App.tsx

```diff
@@ -44,115 +32,160 @@ export default function App() {
   // check localStorage if it contains a - from before - chosen network, if not
   // retrieve the 1st available network from the given ones, else (rare and wrong case)
   // default to polkadot
-  const [chain, setChain] = useState<string>(
-      location ||
-      localNetwork && localNetwork.toLowerCase() ||
-      Object.keys(allChains)[0] ||
-      "polkadot"
+  const currentName =
+    (Object.keys(allChains).includes(location) && location) ||
+    localNetwork ||
+    Object.keys(allChains)[0] ||
+    "polkadot";
+
+  const [currentNetwork, setCurrentNetwork] = useState<string>(currentName);
+  const [metadataQr, setMetadataQr] = useState<QrInfo>(
+    allChains[currentName]?.metadataQr
+  );
+  const [chain, setChain] = useState<ChainSpec>(allChains[currentName]);
+  const [specsQr, setSpecsQr] = useState<QrInfo>(
+    allChains[currentName]?.specsQr
   );
 
-  const specs = allChains[chain];
-  if (!specs){
-    return null
-  }
-  const extraInfo = getChain(chain);
-  document.body.style.backgroundColor = extraInfo?.secColor || "";
+  useEffect(() => {
+    const name = currentNetwork?.toLowerCase();
+    if (name) {
+      setChain(allChains[name]);
+      setMetadataQr(allChains[name]?.metadataQr);
+      setSpecsQr(allChains[name]?.specsQr);
+      // In case the changed name is not the same as the url
+      // then change the url accordingly to the selected network
+      if (name !== location) window.location.assign("#/" + name);
+    }
+  }, [currentNetwork, allChains]);
 
-  const onNetworkSelect = (network: NetworkDetails) => {
-    setChain(network.name.toLowerCase());
-    setLocalNetwork(network.name);
-    window.location.assign("#/" + network.name.toLowerCase())
-  };
+  useEffect(() => {
+    if (isOpen) {
+      setSidebarStyle(
+        "w-64 bg-white px-4 absolute md:left-0 md:h-[91vh] h-[92vh] md:border-r-0 border-r-2 border-neutral-200 z-30 left-0"
+      );
+    } else {
+      setSidebarStyle(
+        "w-64 bg-white px-4 absolute md:left-0 md:h-[91vh] h-[92vh] md:border-r-0 border-r-2 border-neutral-200 z-30 md:z-0 left-[-17rem]"
+      );
+    }
+  }, [isOpen]);
+
+  document.body.style.backgroundColor = "#F5F5F5";
+  const { color } = allChains[currentName] || { color: "#9C9C9C" };
 
-  return (
-      <div className="flex flex-col">
+  return !chain ? (
+    <></>
+  ) : (
+    <div className="flex flex-col bg-white">
+      <div
+        className="md:flex justify-between px-10 py-2 items-center text-xl"
+        style={{ backgroundColor: color }}
+      >
+        <div className="text-white md:w-1 font-bold text-2xl text-left m-auto md:m-0">
+          Metadata Update Portal
```

Comment:
This needs to be on two lines, not on three.

Expected:
<img width="324" alt="image" src="https://user-images.githubusercontent.com/7165377/172149296-6ccc8fac-b19b-49ac-89e6-b361bb5b93d4.png">

Current:
<img width="169" alt="image" src="https://user-images.githubusercontent.com/7165377/172149331-76ad4ec7-f1c0-4b38-8027-07d6860270f7.png">

https://www.figma.com/file/eOnz3xbzObTG8hQlIRs6jg/Metadata-Portal?node-id=0%3A1